lumicks/pylake/tests/conftest.py

```diff
@@ -115,6 +124,51 @@ def h5_file(tmpdir_factory, request):
         mock_file.make_calibration_data("3", "Force 1y", {calibration_time_field: 10})
         mock_file.make_calibration_data("4", "Force 1y", {calibration_time_field: 100})
 
+        counts = [2, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2, 1, 8, 0,
+                  0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 8, 0,
+                  1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 8, 0,
+                  0, 0, 0, 0, 1, 0, 0, 1, 0, 0, 1, 0, 0, 0, 8, 0]
+
+        infowave = [1, 0, 2, 1, 0, 2, 1, 0, 2, 1, 0, 2, 1, 0, 0, 2,
+                    0, 1, 2, 0, 1, 2, 0, 1, 2, 0, 1, 2, 0, 1, 0, 2,
+                    1, 0, 2, 1, 0, 2, 1, 0, 2, 1, 0, 2, 1, 0, 0, 2,
+                    0, 1, 2, 0, 1, 2, 0, 1, 2, 0, 1, 2, 0, 1, 0, 2]
+
+        json_string = '{"value0": {'\
+                          '"cereal_class_version": 1, '\
+                          '"fluorescence": true, '\
+                          '"force": false, '\
+                          '"scan count": 0, '\
+                          '"scan volume": {"'\
+                                'center point (um)": {'\
+                                    '"x": 58.075877109272604, '\
+                                    '"y": 31.978375270573267, '\
+                                    '"z": 0'\
+                                '}, '\
+                                '"cereal_class_version": 1, '\
+                                '"pixel time (ms)": 0.2, '\
+                                '"scan axes": ['\
+                                    '{'\
+                                        '"axis": 0, '\
+                                        '"cereal_class_version": 1, '\
+                                        '"num of pixels": 5, '\
+                                        '"pixel size (nm)": 10, '\
+                                        '"scan time (ms)": 0, '\
+                                        '"scan width (um)": 36.07468112612217'\
+                                    '}'\
+                                ']'\
+                            '}'\
+                        '}'\
+                    '}'
```

Comment:
Wouldn't it be cleaner to create a JSON from dictionary and dump it as a string?

Reply:
Hahaha, yes... I don't know why it didn't occur to me to do it that way actually 🤔

Anyways, it starts its life as a dictionary now.